### PR TITLE
gas-section-five done

### PR DESCRIPTION
**Pull request by BoshraM**

### CODE REVIEW

File: 04-gas/gas-section-five/gas-section-five.html [hunk 1]
```diff
@@ -14,29 +14,24 @@
...
+            </p>
+            <p class="gas-section-five-paragraph">
+              If you're about to come out of a fixed-rate tariff contract, it could be worth considering how to switch energy suppliers or contacting your current supplier to make sure you're on the best tariff for your home's energy usage and comparing it with competitor providers.
+            </p>
+          </div>
         </div>
 
-        <div class="gas-section-five-img-container">
+        <div class="gas-section-five-image-container">
                 <img
```
Comment (bazmurphy) [inline 04-gas/gas-section-five/gas-section-five.html:33]: fix the indentation here

File: 04-gas/gas-section-five/gas-section-five.css [hunk 2]
```diff
@@ -1,59 +1,147 @@
...
+    font-weight: var(--paragraph-font-weight-desktop);
+  }
+}
+
+.gas-section-five-image-container {
+  grid-area: gas-section-five-image-container;
+  display: grid;
+  align-content: start;
+  background-color: var(--level3-background-color);
+  justify-content: center;
```
Comment (bazmurphy) [inline 04-gas/gas-section-five/gas-section-five.css:130]: put these in the order :

justify-content
align-content
background-color

File: 04-gas/gas-section-five/gas-section-five.css [hunk 3]
```diff
@@ -1,59 +1,147 @@
...
+
+@media (min-width: 600px) {
+  .gas-section-five-title {
+    flex-direction: row;
+    column-gap: var(--title-container-column-gap-desktop);
+    row-gap: none;
+    text-align: left;
+    font-size: var(--gas-title-font-size-desktop);
+    font-weight: var(--gas-title-font-weight-desktop);
+  }
```
Comment (bazmurphy) [inline 04-gas/gas-section-five/gas-section-five.css:92]: i assume 
--title-container-row-gap-mobile
--title-container-column-gap-desktop
--gas-title-font- 

etc..

is in your updated global-styles.
i can't see how this looks when applied to check it.

if the title-container rule is specific to gas then maybe you can prefix it with gas

it depends if you have another use-case elsewhere for it

**Review by bazmurphy — CHANGES_REQUESTED** [review]

File: 04-gas/gas-section-five/gas-section-five.html (hunk 1 above)
Comment (BoshraM) [reply 04-gas/gas-section-five/gas-section-five.html:33]: done

File: 04-gas/gas-section-five/gas-section-five.css (hunk 3 above)
Comment (BoshraM) [reply 04-gas/gas-section-five/gas-section-five.css:92]: here they must be different with other page and hero section. for now we can keep like this

File: 04-gas/gas-section-five/gas-section-five.css (hunk 2 above)
Comment (BoshraM) [reply 04-gas/gas-section-five/gas-section-five.css:130]: done

**Pull request merged**